proto/regen/ecocredit/basket/v1beta1/tx.proto

```diff
@@ -4,7 +4,7 @@ package regen.ecocredit.basket.v1beta1;
 
 import "regen/ecocredit/basket/v1beta1/types.proto";
 
-option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/v1beta1";
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1beta1";
```

Comment:
code wasn't being generated for these packages, i assume they were being overwritten by core which gets written last? all code from these sub packages get generated now